region/brinstar/pink/Big Pink.json

```diff
@@ -1147,48 +1165,13 @@
       ],
       "flashSuitChecked": true
     },
-    {
-      "id": 43,
-      "link": [5, 5],
-      "name": "Carry G-Mode Back Through",
-      "entranceCondition": {
-        "comeInWithGMode": {
-          "mode": "direct",
-          "morphed": false
-        }
-      },
-      "requires": [],
-      "exitCondition": {
-        "leaveWithGMode": {
-          "morphed": false
-        }
-      },
-      "bypassesDoorShell": true,
-      "flashSuitChecked": true
-    },
-    {
-      "id": 44,
-      "link": [5, 5],
-      "name": "Carry G-Mode Morph Back Through",
-      "entranceCondition": {
-        "comeInWithGMode": {
-          "mode": "direct",
-          "morphed": true
-        }
-      },
-      "requires": [],
-      "exitCondition": {
-        "leaveWithGMode": {
-          "morphed": true
-        }
-      },
-      "bypassesDoorShell": true,
-      "flashSuitChecked": true
-    },
     {
       "id": 45,
-      "link": [5, 14],
+      "link": [5, 13],
       "name": "Base",
+      "entranceCondition": {
+        "comeInNormally": {}
+      },
```

Comment:
just realized, is this going to have an implicit [5, 5] come in with mockball, that will override this?

Reply:
There would be a [5, 5] implicit comeInWithMockball, yes. I think you'll want to override it by setting `"useImplicitComeInWithMockball": false` and adding a [5, 13] comeInWithMockball strat.

Probably we should add tests that ensure if one of these "useImplicit" properties is overridden then all of them should be, to make sure cases aren't overlooked.